### PR TITLE
Add base doc for snyk

### PR DESCRIPTION
**Pull request by louislang**
## Checklist

- [ ] Does this PR have an associated issue (i.e., `closes #<issueNum>` in description above)?
- [ ] Have you ensured that you have met the expected acceptance criteria?

### CODE REVIEW

File: docs/integrations/snyk.md
Comment (kylewillmon) [inline docs/integrations/snyk.md:1]: Find and replace `snky` with `snyk`...

I've made that typo so many times during this integration... It sticks out when I see it now 😉

Comment (louislang) [reply docs/integrations/snyk.md:1]: 🤦 fixed

Comment (maxrake) [inline docs/integrations/snyk.md:1]: This new integration should be added to the [Integrations Overview](https://docs.phylum.io/docs/integrations_overview) page.

**Review by maxrake — CHANGES_REQUESTED** [review]
One request to add this new integration to the [Integrations Overview](https://docs.phylum.io/docs/integrations_overview) page.

**Review by maxrake — APPROVED** [review]
LGTM

**Pull request merged**